tasmanian/utils/plot.py

```diff
@@ -31,12 +31,15 @@ def table_to_json(df, normalize=True):
         dfc = df.copy()
         errors = {'a': a, 'c': c, 'g': g, 't': t}
         
+        mutations_list = list(mutations)
+        
         if normalize:
-            for i in errors.keys():
-                dfc.loc[:, errors[i]] /= (dfc.loc[:, errors[i]].sum(axis=1).values.reshape(-1,1) + 10e-6 )
-            
-        table_max = dfc[mutations].max().max() * 1.1
-        table_min = dfc[mutations].min().min() * 0.5
+            dfc[mutations_list] = dfc[mutations_list].div(dfc[mutations_list].sum(axis=1), axis=0)
+            # Use newer NumPy syntax for type conversion
```

Comment:
The addition of a very small number (1e-6) is probably to avoid dividing by zero.